nistats/regression.py

```diff
@@ -330,3 +329,65 @@ def SSE(self):
     def MSE(self):
         """ Mean square (error) """
         return self.SSE / self.df_resid
+
+
+class SimpleRegressionResults(LikelihoodModelResults):
```

Comment:
We'll probably have to refactor this later, but for the moment I'm fine with it as is.